src/SimpleEventStore/SimpleEventStore.AzureDocumentDb.Tests/AzureDocumentDBEventStoreInitializing.cs

```diff
@@ -81,13 +81,15 @@ public async Task when_using_shared_throughput_it_is_set_at_a_database_level()
                     collectionOptions.CollectionName = collectionName;
                     collectionOptions.CollectionRequestUnits = null;
                 },
-                databaseOptions => { databaseOptions.DatabaseRequestUnits = throughput; });
+                dbThroughput);
```

Comment:
Is there not value in leaving the databaseOptions?  The original intent with the collectionOptions was to provide an extensibility point so we could add new settings easily without breaking consumers, so that's why we added the databaseOptions class.  It's just meant to be a limited sub-set of what settings we'd want people to have control over.  If we don't want to expose certain options, we wouldn't.

Feels like we should be consistent here with the collection options.

Reply:
Ok sure, that make sense. But, with this in mind I would propose changing the name to be `UseDatabase` rather than the more specific `UseSharedThroughput`.

This would make more sense when configure the database and bring it more inline with the rest of the builder methods such as `UseCollection` and `UseLogging` that update the other storage engine options

Reply:
@asosMikeGore This change has been made and pushed

Reply:
Thanks.  Agree UseDatabase is better!